### PR TITLE
Add logos

### PR DESCRIPTION
**Pull request by Smaug123**
After a `dotnet pack`:

Screenshot 2024-04-15 at 21 58 17

I don't know how one usually deals with transparent logos given the existence of light mode and dark mode. I have GitHub in dark mode, which means the white logo is best; but I used VS Code to visualise the Markdown render, and that had a white background so the dark logo was best.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # ApiSurface
 
+<img src="logos/text-white.png" alt="ApiSurface logo: a minimalistic cube, dimetric projection, composed of three stacked layers which are joined at some edges so as to suggest the letter S. The visible parts of the stacked layers inside the cube are light blue; the outer surface is white. To the right is the text 'ApiSurface' in the same colour as the cube's exterior." style="height:200px;object-fit:cover"/>
```
Comment (zuqq) [inline README.md:3]: I think that the logo would look better if it were cropped to its actual dimensions; in the preview that GitHub shows me there's a lot of dead space.

Comment (nojaf) [reply README.md:3]: You indeed want to try the `picture` tag here, the preview shows

image


for me.

**Comment by Smaug123** [comment]
Thanks, that's fixed the light- and dark-mode problem. Just need to crop the images now.

**Comment by nojaf** [comment]
Yup, works for me now:

image

**Comment by Smaug123** [comment]
Cropped the original 1080x1080 images to dimensions width=1080*7/8, height=1080/4.

**Comment by Smaug123** [comment]
Screenshot 2024-05-01 at 22 39 51
Screenshot 2024-05-01 at 22 42 00

**Review by bward — APPROVED** [review]
pretty!

**Pull request merged**